flake8_quotes/__init__.py

```diff
@@ -223,19 +235,38 @@ def get_quotes_errors(self, file_contents):
                 }
             # Otherwise (string is inline quote)
             else:
-                # If our string is a known good string, then ignore it
-                #   (')foo' -> good (continue)
-                #   "it(')s" -> good (continue)
-                #   (")foo" -> possibly bad
-                if self.config['good_single'] in unprefixed_string:
+                #   'This is a string'       -> Good
+                #   'This is a "string"'     -> Good
+                #   'This is a \"string\"'   -> Good
+                #   'This is a \'string\''   -> Bad (Q004)  Escaped inner quotes
```

Comment:
It looks like we say Q004 here but it's actually Q003, I'll correct this before landing (same with other Q004 references)